tests/integration/llm-judge.test.ts

```
@@ -0,0 +1,716 @@
+      actualResponses,
+      expectedToolCalls: conversation.expectedToolCalls,
+      actualToolCalls: finalToolCalls,
+      evaluation: {
+        ...evaluation,
+        averageScore
+      },
+      passed,
+      responseTime: totalResponseTime
+    };
+  }
+}
```

Biome (2.1.2)

[error] Do not export from a test file.



(lint/suspicious/noExportsInTest)